test/app/presenters/track_test.rb

```diff
@@ -6,21 +6,34 @@
 class PresentersTrackTest < Minitest::Test
   def setup
     @track = ExercismWeb::Presenters::Track.new("test_track")
+
+    trackler_tracks = Object.new
     @trackler_track = Object.new
-    @track.stubs(:trackler_track).returns(@trackler_track)
-    @track.stubs(:fallback_topic_content).returns("Default topic content")
+
+    Trackler.stubs(:tracks).returns(trackler_tracks)
+    trackler_tracks.stubs(:[]).returns(@trackler_track)
+    @trackler_track.stubs(:repository).returns("https://github.com/exercism/test_track")
+    @trackler_track.stubs(:doc_format).returns("md")
```

Comment:
Minor thing: I'd put these 2 lines (15-16) directly under line 11 so the @trackler_track is fully initialised before you try to use it anywhere.